### ISSUE TITLE
can we set settings ovet wp-xli? 

### ISSUE THREAD

**Issue by osevan** [rtCamp/nginx-helper#307]
We should set settings easy without dashboard login for nginx-helper,  especially,  when you have several sites. 

Which tables should i edit to fix this manually with sql update command? 

Because wp-cli settings integration isnt possible for now. 

Thanks and

Best regards

**Comment by sistason** [comment]
I think you mean how can we **initially** set the default settings? Because `wp option patch update rt_wp_nginx_helper_options enable_purge 1` is working fine with `wp-cli`.

But until you set the default settings, which seems to only be possible by clicking a setting (and therefore merging it with the defaults), the db (and therefore wp-cli) is emtpy.

My solution would be to write the default settings to the options table when activating the plugin. Can this be a feature for the future?

**Comment by osevan** [comment]
> I think you mean how can we **initially** set the default settings? Because `wp option patch update rt_wp_nginx_helper_options enable_purge 1` is working fine with `wp-cli`.
> 
> But until you set the default settings, which seems to only be possible by clicking a setting (and therefore merging it with the defaults), the db (and therefore wp-cli) is emtpy.
> 
> My solution would be to write the default settings to the options table when activating the plugin. Can this be a feature for the future?

Yes. I hope new feature for wp-cli for settings setting ober command line could be available soon,  for full automized settings setup.

**Comment by osevan** [comment]
Any information about this?
